src/app/form/(formSteps)/personal-information/page.tsx

```diff
@@ -1,11 +1,460 @@
+                    <TextInput
+                      id="city"
+                      type="text"
+                      required
+                      validationStatus={errors.city ? "error" : undefined}
+                      aria-invalid={errors.city ? "true" : "false"}
+                      aria-describedby={errors.city && "cityErrorMessage"}
+                      {...register("city", {
+                        required: `${orderedInputNameToLabel["city"]} is required`,
+                      })}
+                    />
+                    {errors.city && (
+                      <span id="cityErrorMessage" className="usa-error-message">
+                        {errors.city.message}
+                      </span>
+                    )}
+                  </div>
+                </div>
+                <div className="grid-row grid-gap">
+                  <div className="mobile-lg:grid-col-6">
+                    <Label htmlFor="state" requiredMarker>
+                      {orderedInputNameToLabel["state"]}
+                    </Label>
+                    <Select className="usa-select" id="state" required {...register("state")}>
+                      {Object.keys(AddressState).map((state) => (
+                        <option key={state} value={state}>
+                          {state}
+                        </option>
+                      ))}
+                    </Select>
+                  </div>
+                  <div className="mobile-lg:grid-col-4">
+                    <Label htmlFor="zip" requiredMarker>
+                      {orderedInputNameToLabel["zip"]}
+                    </Label>
+                    <TextInputMask
+                      className="usa-input--medium"
+                      id="zip"
+                      type="text"
+                      value={zip ?? ""}
+                      mask="#####"
+                      pattern="\d{5}"
+                      required
+                      validationStatus={errors.zip ? "error" : undefined}
+                      aria-invalid={errors.zip ? "true" : "false"}
+                      aria-describedby={errors.zip && "zipErrorMessage"}
+                      {...register("zip", {
+                        required: `${orderedInputNameToLabel["zip"]} is required`,
+                        minLength: {
+                          value: 5,
+                          message: `${orderedInputNameToLabel["zip"]} must have five digits`,
+                        },
+                      })}
+                    />
+                    {errors.zip && (
+                      <span id="zipErrorMessage" className="usa-error-message">
+                        {errors.zip.message}
+                      </span>
+                    )}
+                  </div>
+                </div>
+              </Fieldset>
+            </div>
+            <div className="flex-1">
+              <div>
+                <h3 className="font-heading-md">FAQ</h3>
```

Comment:
This H3 is currently nested under mailing address, however, it belong under the "shop experience" h2